ts/screens/wallet/payment/TransactionErrorScreen.tsx

```diff
@@ -114,7 +118,18 @@ const requestZendeskAssistanceForPaymentFailure = (
     zendeskBlockedPaymentRptIdId,
     RptIdFromString.encode(rptId)
   );
+  // Add organization fiscal code custom field
+  addTicketCustomField(
+    zendeskPaymentOrgFiscalCode,
+    rptId.organizationFiscalCode
+  );
+  // Add rptId custom field
+  addTicketCustomField(zendeskPaymentNav, getCodiceAvviso(rptId));
   if (payment) {
+    // Add failure custom field
+    addTicketCustomField(zendeskPaymentFailure, payment.failure as string);
```

Comment:
As above.

Reply:
solved in [a7ea040](https://github.com/pagopa/io-app/pull/4431/commits/a7ea0408ac4eda03089d9c1d11fb8a9ad0e9b595)